### DISCUSSION TITLE
Issue Certificate Error

### DISCUSSION THREAD

**Discussion by voyager163** [polymind-inc/acmebot#312] [Q&A] [answered]
Hi Shibayan,

Can i check if the below error is due to the ACME Rate limits? or it is because I disable the storage settings "Allow Blob public access"?

IssueCertificate

Microsoft.Azure.WebJobs.Extensions.DurableTask.FunctionFailedException: The activity function 'Order' failed: "Failed to deserialize exception from TaskActivity: {"$type":"ACMESharp.Protocol.AcmeProtocolException, ACMESharp","ProblemType":0,"ProblemTypeRaw":null,"ProblemDetail":null,"ProblemStatus":-1,"StackTrace":"   at ACMESharp.Protocol.AcmeProtocolClient.SendAcmeAsync(Uri uri, HttpMethod method, Object message, HttpStatusCode[] expectedStatuses, Boolean skipNonce, Boolean skipSigning, Boolean includePublicKey, CancellationToken cancel, String opName)\r\n   at ACMESharp.Protocol.AcmeProtocolClient.GetNonceAsync(CancellationToken cancel)\r\n   at KeyVault.Acmebot.Internal.AcmeProtocolClientFactory.CreateClientAsync() in /home/runner/work/keyvault-acmebot/keyvault-acmebot/KeyVault.Acmebot/Internal/AcmeProtocolClientFactory.cs:line 56\r\n   at KeyVault.Acmebot.Functions.SharedActivity.Order(IReadOnlyList`1 dnsNames) in /home/runner/work/keyvault-acmebot/keyvault-acmebot/KeyVault.Acmebot/Functions/SharedActivity.cs:line 113\r\n   at Microsoft.Azure.WebJobs.Host.Executors.FunctionInvoker`2.InvokeAsync(Object instance, Object[] arguments) in C:\\projects\\azure-webjobs-sdk-rqm4t\\src\\Microsoft.Azure.WebJobs.Host\\Executors\\FunctionInvoker.cs:line 52\r\n   at

![Screenshot 2021-04-26 at 11 16 27 PM](https://user-images.githubusercontent.com/4599055/116107234-6e1c4480-a6e5-11eb-805b-5d386520b739.png)

Best Regards,
Jonathan Ng

**Comment by shibayan** [comment, marked as answer]
It is probably ACME Rate Limits, and the Storage setting is irrelevant.

**Reply by voyager163** [reply]
Hi Shibayan,

I already screenshot the new Authentication page for the app service, how can i help u update the README.
